client/jetpack-onboarding/steps/business-address.jsx

```diff
@@ -98,9 +98,21 @@ class JetpackOnboardingBusinessAddressStep extends React.PureComponent {
 									required={ fieldName !== 'state' }
 									value={ this.state[ fieldName ] }
 								/>
+								{ fieldName !== 'state' && (
+									<FormInputValidation
```

Comment:
I'm a little hesitant to display a success validation message if everything is fine. See https://github.com/Automattic/wp-calypso/pull/21582#issuecomment-361070274.

Reply:
Good point. Let's wait for the design feedback and if needed address it in both forms.